src/F2_autoreport.py

```diff
@@ -279,6 +320,44 @@ def insert_log_messages(log_dict):
 def save_plots_to_disk(
     fig_obj, file_name, file_path_dict, width=None, height=None, scale=None
 ):
+    r"""
+    This function saves the plots generated using the Plotly library in this module to the outputs folder.
+
+    Parameters
+    ----------
+    fig_obj:
```

Comment:
what is the type of fig_obj

Reply:
its an object/instance of the classes in plotly go library according to [this](https://plotly.com/python/graph-objects/)

Reply:
You should then also write that after the ":" ;)

Reply:
done